stellar-core-postgres/stellar-core_postgres.cfg

```diff
@@ -12,30 +12,194 @@ DATABASE="postgresql://dbname=stellar user=stellar"
 CATCHUP_COMPLETE=false
 CATCHUP_RECENT=1024
 
-UNSAFE_QUORUM=true
-FAILURE_SAFETY=1
+# IMPORTANT
+# You need to change domains and validators below to fit your needs
 
 [[HOME_DOMAINS]]
-HOME_DOMAIN="testnet.stellar.org"
+HOME_DOMAIN="www.stellar.org"
```

Comment:
Are you switching to pubnet on purpose?

Reply:
oh didn't realize this was a testnet file! fixing...